### ISSUE TITLE
add warning to functions in cpu.py

### ISSUE THREAD

**Issue by giulianogiari** [MattiaSilvestri/energy-monitor#17]

**Comment by giulianogiari** [comment]
this referes to get_cpu_info and get_cpu_usage. 
for the missing tdp data we decided to throw an error rather than a warning. do we want to do the same for the other functions?
i guess that get_cpu_info yes, because the code will anyway throw errors later, whereas for cpu_percentage since it will be called multiple times during we can send a warning to not break the code if it doesnt return the expected output just once?

**Comment by FrnRmn** [comment]
Yep, get_cpu_info() should raise a specific error of its own. Currently, if it fails, it will make get_cpu_tdp() fail, without discerning why.
